examples/with-i18n-next-intl/components/Navigation.js

```diff
@@ -0,0 +1,26 @@
+import { useTranslations } from 'next-intl'
+import { useRouter } from 'next/dist/client/router'
+import Link from 'next/link'
+
+export default function Navigation() {
+  const t = useTranslations('Navigation')
```

Comment:
So is this loading all the translations? Or just the current one?

Reply:
These messages are now provided through the shared messages, which don't contain any page-specific translations.